lib/mix/lib/mix/tasks/release.ex

```diff
@@ -622,6 +622,13 @@ defmodule Mix.Tasks.Release do
       files to. It can be set to a custom directory. It defaults to
       `$RELEASE_ROOT/tmp`
 
+    * `RELEASE_DISTRIBUTION` - how do we want to run the distribution.
+      Using `name` (long names) or `sname` (short names). Defaults to
+      `sname` which allows access only within the current system.
+      `name` allows external connections. If `name` is used and you are
```

Comment:
Might be worth making it clear that `name` requires a hostname that resolves through DNS or an ip. While short name you can pretty much give anything, as long as it doesn't have a `.` in it.